WooCommerce/WooCommerceTests/ViewRelated/Dashboard/Settings/In-Person Payments/Deposits Overview/WooPaymentsDepositsCurrencyOverviewViewModelTests.swift

```diff
@@ -0,0 +1,57 @@
+import XCTest
+@testable import WooCommerce
+
+final class WooPaymentsDepositsCurrencyOverviewViewModelTests: XCTestCase {
+
+    var sut: WooPaymentsDepositsCurrencyOverviewViewModel!
+    var analyticsProvider: MockAnalyticsProvider!
```

Comment:
```suggestion
    private var sut: WooPaymentsDepositsCurrencyOverviewViewModel!
    private var analyticsProvider: MockAnalyticsProvider!
```

Reply:
I don't think we need to do this; it's a test class. We could equally mark any helpers and even test methods as private (though that might hide them from the test runner, I'm not sure how that works). The interface of a test class just isn't that important to add extra words, IMO